examples/basic-example/README.md

```diff
@@ -37,3 +57,5 @@ yarn hardhat deploy-zksync
 - `yarn hardhat compile`: compiles all the contracts in the `contracts` folder.
 - `yarn hardhat deploy-zksync`: runs all the deploy scripts in the `deploy` folder.
     - To run a specific script, add the `--script` argument, e.g. `--script 001_deploy.ts`.
+    - To run on a specific zkSync network, add the `--zksyncNetwork` argument, e.g. `--zksyncNetwork zkTestnet`
```

Comment:
couldn't this use the default --network? given it has the zkSync:true?

Reply:
looking at the deployer code, we can indeed... what is the benefit of having the zkSyncNetwork in this case?

Reply:
Good point.
Originally, the reason was to more easily handle the default scenario when user doesn't specify the network, because in this case hre.network will be set to 'hardhat' network by hardhat. Since for zkSync deploy 'hardhat' network is not an option we want to set 'localhost' as the default option. 
However, it is still possible to use standard hardhat --network argument for this, so I updated the PR.